aiohttp/test_utils.py

```diff
@@ -65,8 +66,11 @@ def get_unused_port_socket(host: str) -> socket.socket:
     return get_port_socket(host, 0)
```

Comment:
Should this helper also support the family arg?

Reply:
Can't hurt. Added.